src/components/team/styledTeam.js

```diff
@@ -0,0 +1,61 @@
+import styled from "styled-components";
+
+export const TeamWrapper = styled.section`
+    display: flex;
+    flex-direction: column;
+    width: 53.61%;
+    margin: auto;
+    padding: 150px 0;
+`
+export const TeamBlocks = styled.div`
+    display: flex;
+    justify-content: space-between;
+    margin: 80px auto 120.8px auto;
+    width: 100%;
+`
+export const TeamBlock = styled.div`
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+`
+export const TeamImage = styled.img`
+    width: 146px;
+    height: 146px;
+    margin-bottom: 20px;
+`
+export const TeamText = styled.p`
+    font-style: normal;
+    font-weight: normal;
+    font-size: 24px;
+    line-height: 28px;
+    text-align: center;
+`
+export const Link = styled.a`
```

Comment:
You should be able to grab the entire linkedin icon and combine it into one element. It would remove the need for alot of the code here. I can show you how if you want. It takes a few clicks in figma